.dockerignore

```diff
@@ -0,0 +1,6 @@
+.vscode
+.git
+bazel-out
+bazel-arcs
+bazel-bin
+
```

Comment:
Extraneous blank line. Ignore if this is just a copy of a sample file.

Reply:
Done.